lib/MusicBrainz/Server/Data/ReleaseLabel.pm

```diff
@@ -136,7 +136,6 @@ sub insert
 
     my $release_id = $row->{release};
     $self->c->model('Series')->reorder_for_entities('release', $release_id);
-    $self->c->model('Release')->_delete_from_cache($release_id);
```

Comment:
Does having this in preparation for that time when we get more intelligent invalidation logic actually hurt?

Reply:
I don't know if/when that will happen, so it does hurt *slightly* in that it will increase the number of release cache misses for no benefit.

Reply:
Oh, ok. I thought we'd want stuff to be uncached when it changes anyway, that's why I asked.

Reply:
Just to be clear, we do, but in this case nothing is changing anymore. :) (Because release labels are no longer cached on the release.)